libs/sdk-ui-tests/tests/smoke-and-capture/allScenarios.test.tsx

```diff
@@ -89,7 +89,6 @@ function scenarioSaveDefinition(
 
     writeAsJsonSync(path.join(recordingDir, RecordingFiles.Execution.Definition), {
         ...triggeredExecution,
-        buckets: [],
```

Comment:
This solves the issue with recordings. Buckets were omitted from stored definitions.

Note, that the object still must be copied and cannot be used as is, otherwise the jest test fails. It seems this function modifies it and it messes up the flow in later stages.